azure/packages/test/scenario-runner/README.md

```diff
@@ -1,8 +1,50 @@
 # @fluidframework/azure-scenario-runner
 
+## Azure Scenario Runner
+
+This package provides a reference application that executes pre-set scenarios with the intent of measuring the Fluid Framework's performance and reliability. We primarily use this package as part of a pipeline scheduled to run periodically to measure, log, and report various performance and reliability metrics. These metrics can then be used to gauge an undrestanding of the expected behavior/performance of the Fluid Framework in these various scenarios, which can help define the SLA.
+
+## Implemented Scenarios
+
+### Azure Client
+
+Tests creating an Azure Client
+
+### Doc Creator
+
+This scenario creates a bunch of empty Fluid documents and measures the time it takes to create theses documents.
+
+### Doc Loader
+
+This scenario loads a set of previously created docs multiple times and measures the time it takes to load these documents.
+
+### Map Traffic
+
+[TBD]
```

Comment:
Could we please add the description here.

Reply:
This PR doesn't utilize Map Traffic yet. Should we still describe that, because I assume we'll have another PR for enabling and extending the MapTraffic test